index.rst

```diff
@@ -1,37 +1,55 @@
-EPICS Documentation
+Welcome to the EPICS Documentation!
```

Comment:
I got this WARNING:

```
^^^^^^^^^^^^^^^^^^^^^
/home/gabrielfedel/events/EPICSDocumentathon2023/epics-docs/index.rst:2: WARNING: Title underline too short.

Welcome to the EPICS Documentation!
===================

```

Reply:
Ops, I missed that

Reply:
I think this is not fixed yet

Reply:
Ah, looks I committed by mistake not what I wanted